### PR TITLE
Release `1.0.0-pre1`

### PR DESCRIPTION
**Pull request by dmitrykuzmin**
This PR upgrades `jdbc-storage` version as well as `base` and `core` dependencies to `1.0.0-pre1`.

Notable changes:

1. `StandStorage` functionality was removed.
2. `JdbcRecordStorage.readMultiple(ids, fieldMask)` method now obeys it's contract returning `null` for all missing or inactive entity IDs.

### CODE REVIEW

**Comment by dmitrykuzmin** [comment]
@dmdashenkov PTAL.

File: rdbms/src/main/java/io/spine/server/storage/jdbc/record/JdbcRecordStorage.java [hunk 1]
```diff
@@ -112,10 +114,16 @@ public boolean delete(I id) {
...
-    protected Iterator<EntityRecord> readMultipleRecords(Iterable<I> ids,
-                                                         FieldMask fieldMask) {
-        Iterator<EntityRecord> records = table.readByQuery(toQuery(ids), fieldMask);
-        return records;
+    protected Iterator<@Nullable EntityRecord> readMultipleRecords(Iterable<I> ids,
+                                                                   FieldMask fieldMask) {
+        Collection<EntityRecord> records = new ArrayList<>();
+        for (I id : ids) {
+            EntityQuery<I> query = toQuery(id);
+            EntityRecord recordOrNull = readRecordOrNull(query, fieldMask);
```
Comment (dmdashenkov) [inline rdbms/src/main/java/io/spine/server/storage/jdbc/record/JdbcRecordStorage.java]: This seems rather inefficient. We should keep the number of calls to the DB as low as possible.

File: rdbms/src/test/java/io/spine/server/storage/jdbc/aggregate/MultipleExceptionsOnCloseTest.java [hunk 2]
```diff
@@ -46,16 +47,9 @@ void provideToString() {
...
-                       aggregatingException.toString());
-        assertContains(stringName,
-                       ex2.toString(),
-                       aggregatingException.toString());
-        assertContains(stringName,
-                       ex3.toString(),
-                       aggregatingException.toString());
+        assertThat(aggregatingException.toString(), containsString(ex1.toString()));
+        assertThat(aggregatingException.toString(), containsString(ex2.toString()));
+        assertThat(aggregatingException.toString(), containsString(ex3.toString()));
```
Comment (dmdashenkov) [inline rdbms/src/test/java/io/spine/server/storage/jdbc/aggregate/MultipleExceptionsOnCloseTest.java]: We might as well extract `aggregatingException.toString()` into a variable.

**Comment by dmitrykuzmin** [comment]
@dmdashenkov PTAL.

File: rdbms/src/main/java/io/spine/server/storage/jdbc/aggregate/SelectEventRecordsById.java [hunk 3]
```diff
@@ -63,9 +63,12 @@ private SelectEventRecordsById(Builder<I> builder) {
                                                   .setFetchSize(fetchSize)
                                                   .build());
         ResultSet resultSet = query.getResults();
-        return new MessageDbIterator<>(resultSet,
-                                       AGGREGATE.name(),
-                                       AggregateEventRecord.getDescriptor());
+
+        DbIterator<AggregateEventRecord> dbIterator = DbIterator.createFor(
+                resultSet,
+                messageColumn(AGGREGATE.name(), AggregateEventRecord.getDescriptor())
```
Comment (dmdashenkov) [inline rdbms/src/main/java/io/spine/server/storage/jdbc/aggregate/SelectEventRecordsById.java]: Please extract the value into a variable.

File: rdbms/src/main/java/io/spine/server/storage/jdbc/query/DbIterator.java [hunk 4]
```diff
@@ -172,4 +206,43 @@ public void close() throws DatabaseException {
     private static NoSuchElementException noSuchElement() {
         throw new NoSuchElementException("No elements remained.");
     }
+
+    private static class SingleColumnIterator<R> extends DbIterator<R> {
```
Comment (dmdashenkov) [inline rdbms/src/main/java/io/spine/server/storage/jdbc/query/DbIterator.java]: Please document nested classes.

File: rdbms/src/main/java/io/spine/server/storage/jdbc/query/PairedValue.java [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+ * A combination of two values for iterating over the SQL query results.
+ *
+ * <p>Nulls are not accepted as values for the usage convenience.
+ *
+ * @param <A>
+ *         the first value type
+ * @param <B>
+ *         the second value type
+ */
+public final class PairedValue<A, B> {
```
Comment (dmdashenkov) [inline rdbms/src/main/java/io/spine/server/storage/jdbc/query/PairedValue.java]: Can we have a more domain-specific definition? Say, `EntityRecordWithId`, basing on the visible usage?

**Comment by dmitrykuzmin** [comment]
@dmdashenkov PTAL.

File: rdbms/src/main/java/io/spine/server/storage/jdbc/query/EntityRecordWithId.java [hunk 6]
```diff
@@ -20,42 +20,40 @@
...
 
+import io.spine.server.entity.EntityRecord;
+
 import static com.google.common.base.Preconditions.checkNotNull;
 
 /**
- * A combination of two values for iterating over the SQL query results.
- *
- * <p>Nulls are not accepted as values for the usage convenience.
+ * A combination of {@link EntityRecord} and its ID for iterating over the SQL query response.
```
Comment (dmdashenkov) [inline rdbms/src/main/java/io/spine/server/storage/jdbc/query/EntityRecordWithId.java]: Please add English articles here and check through the change set.

**Pull request merged**